### ISSUE TITLE
Chance shards not highlighted

### ISSUE THREAD

**Issue by GhostTales** [NeverSinkDev/NeverSink-Filter-for-PoE2#9]
noticed this on version 0.1.1 of the filter.

![image](https://github.com/user-attachments/assets/d0e71f8f-7e6e-4240-b0a2-1127d753ec9a)

**Comment by GhostTales** [comment]
did a quick fix of adding chance shards to line 158 of filter

```
BaseType "Arcanist's Etcher" "Armourer's Scrap" "Blacksmith's Whetstone" "Orb of Augmentation" "Orb of Transmutation" "Regal Shard" "Chance Shard"
SetTextColor 255 207 132
SetBorderColor 255 207 132
MinimapIcon 2 Grey Circle
```

![image](https://github.com/user-attachments/assets/d756cf49-aec1-42c0-9bb5-737a7ce7f81b)

**Comment by ShyRed** [comment]
Is fixed by #6

**Comment by bratzie** [comment]
I added chance shards and orbs of chance in #6 , just update to the latest version and it should work!
